is_core/generic_views/table_views.py

```diff
@@ -41,6 +41,7 @@ class TableViewMixin(object):
     menu_group_pattern_name = None
     render_actions = True
     enable_columns_manager = False
+    field_headers = None
```

Comment:
nacpal bych to i do core. Navíc by bylo dobré, aby bylo možné měnit jednotlivé labely i podle requestu (už jsme na to narazili na sečtenu), takže si to představuju jako metodu get_foo_label(model, request)